### PR TITLE
Added a Skeleton folder to the components folder that contains our lo…

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**
…ads before getting data and added a condition to the MessagesList so that no empty messages are sent.

### CODE REVIEW

File: screens/Messages/Message.jsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ const Message = ({ isLeft, message }) => {
                     <Text style={[!isLeft ? styles.messageisRight : styles.messageIsLeft]}>{message}</Text>
                 </View>
             </View>
+           {isLeft ? <View style={styles.leftArrow}/> : <View style={styles.rightArrow}/>}
```
Comment (M7Arman) [inline screens/Messages/Message.jsx:13]: `<View style={ isLeft ? styles.leftArrow : styles.rightArrow}/>` better to write like this

File: screens/Messages/MessagesList.jsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ import styles from './style';
 
 const MessagesList = () => {
   const [value, setValue] = useState('');
+  const [sendMessages, setSendMessages] = useState(false);
```
Comment (M7Arman) [inline screens/Messages/MessagesList.jsx:16]: it's not a good practice to store different types in the same variable. Here you store a boolean in one case and an object in another case. Remind me to discuss this tomorrow if you don't see any good solution here

File: screens/Messages/MessagesList.jsx [hunk 3]
```diff
@@ -37,17 +38,24 @@ const MessagesList = () => {
   ]);
 
   useEffect(() => {
-    setValue('');
-  }, [messages.length]);
+    if (value.length > 0 && value.trim() !== '') {
+      setSendMessages(true);
+    } else {
+        setSendMessages(false)
+    }
```
Comment (M7Arman) [inline screens/Messages/MessagesList.jsx:45]: you can write this in one line, like this:

`setSendMessages(value.length > 0 && value.trim() !== '');`

File: screens/Messages/style.js [hunk 4]
```diff
@@ -103,6 +102,26 @@ const styles = StyleSheet.create({
...
+  rightArrow: {
+    position: 'absolute',
+    backgroundColor: '#5DB075',
+    width: 20,
+    height: 30,
+    bottom: 0,
+    borderBottomLeftRadius: 25,
+    right: 20,
+    zIndex: -1,
+  },
```
Comment (M7Arman) [inline screens/Messages/style.js:124]: if 90% of the styles are the same as in your case for `leftArrow` and `rightArrow`, it's better to have one common style(e.g. `arrow`) and then in `leftArrow` and `rightArrow` store the differences. Please update

File: screens/Profile/Profile.jsx [hunk 5]
```diff
@@ -7,15 +7,19 @@ import Posts from './page/Posts/Posts';
...
+import {useEffect} from 'react';
 
 const Profile = ({navigation}) => {
   const [showHide, setShowHide] = useState(false);
   const [imageUrl, setImageUrl] = useState('');
   const {feedData} = useContext(DNAdataContext);
   const [loading, setLoading] = useState(true);
 
+  useEffect(() => {
+    setTimeout(() => setLoading(false), 2500);
```
Comment (M7Arman) [inline screens/Profile/Profile.jsx:20]: if it's a temporary solution until the main logic will be implemented please add TODO comment with the details. If no, please explain why you need this timeout?

File: screens/Messages/Message.jsx (hunk 1 above)
Comment (MiqayelChaloyan) [reply screens/Messages/Message.jsx:13]: Updated.

File: screens/Messages/MessagesList.jsx (hunk 2 above)
Comment (MiqayelChaloyan) [reply screens/Messages/MessagesList.jsx:16]: Okay, will discuss tomorrow.

File: screens/Messages/MessagesList.jsx (hunk 3 above)
Comment (MiqayelChaloyan) [reply screens/Messages/MessagesList.jsx:45]: Have changed.

File: screens/Messages/style.js (hunk 4 above)
Comment (MiqayelChaloyan) [reply screens/Messages/style.js:124]: Updated as mentioned.

File: screens/Profile/Profile.jsx (hunk 5 above)
Comment (MiqayelChaloyan) [reply screens/Profile/Profile.jsx:20]: Added a `TODO` comment.

**Pull request merged**